modules/test/playwright/tests/portal-web/html/common/themes/bottomJS.spec.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../../../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../../../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../../../../fixtures/loginTest';
+import {masterPagesPagesTest} from '../../../../../fixtures/masterPagesPagesTest';
+import {pageEditorPagesTest} from '../../../../../fixtures/pageEditorPagesTest';
+import {pageSelectorPagesTest} from '../../../../../fixtures/pageSelectorPagesTest';
+import {pagesAdminPagesTest} from '../../../../../fixtures/pagesAdminPagesTest';
+import {systemSettingsPageTest} from '../../../../../fixtures/systemSettingsPageTest';
+import getRandomString from '../../../../../utils/getRandomString';
+import {waitForAlert} from '../../../../../utils/waitForAlert';
+
+const test = mergeTests(
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	isolatedSiteTest,
+	loginTest(),
+	masterPagesPagesTest,
+	pageEditorPagesTest,
+	pageSelectorPagesTest,
+	pagesAdminPagesTest,
+	systemSettingsPageTest
+);
+
+const sampleJS = 'window.onload=function(){console.log("customjssnippets")}';
+
+test(
+	'Check JS injection works correctly',
+	{tag: '@LPD-37109'},
+	async ({apiHelpers, page, pageEditorPage, site}) => {
+		await test.step('Add JS to page and check if injected correctly', async () => {
```

Comment:
There is no need for just one step, that just describes the same thing as the test itself. If you wish, you can divide test into several logical steps, but if you prefer to have it all in one go, removing the step wrapper is perfectly fine.

Reply:
Got it, made it one go.